### ISSUE TITLE
Apply Master

### ISSUE THREAD

**Issue by bpascher** [heavysixer/node-pptx#49]
Hey,

is there a possibility to apply a slide master?
In my case, I have an existing .pptx-file and I want to change the slide master of an added slide (considering there is more than one slide master).

**Comment by heavysixer** [comment]
@gregdolley do we have an example of how this might work with the API?

**Comment by gregdolley** [comment]
@heavysixer @bpascher Unfortunately, editing slide masters (whether on an existing slide or not), is still not possible. We've had this feature on our backlog for a long time. Mark - maybe time to revisit it?

**Comment by heavysixer** [comment]
Yeah we should take up this feature again, @bpascher PRs are greatly appreciated if you want to try and take a crack at it.

**Comment by makopyants** [comment]
Hi guys, I will take a look at this feature and maybe produce PR after sometime. If there's anyone who could point me in right direction to get this done - would be greatly appreciated
